spec/models/lti_configuration_spec.rb

```diff
@@ -14,7 +14,7 @@
 
     it "returns the default name when unspecified type is :other" do
       lti_configuration.lms_type = :other
-      expect(lti_configuration.lms_name).to eq("Other Learning Management System")
+      expect(lti_configuration.lms_name).to eq("Other learning management system")
```

Comment:
This test will need to be updated if we do change "Other learning management system" to "other learning management system"